Deployment/SetEntityInTableStorage.ps1

```diff
@@ -0,0 +1,28 @@
+param (
+    [Parameter(Mandatory = $true)] [string] $Strorageaccountname,
```

Comment:
Typo in parameter name 'Strorage', appears a few times so search and replace across the repo

Reply:
suggested changes are implemented.